src/player.cpp

```diff
@@ -4627,7 +4627,7 @@ void player::update_stamina( int turns )
     float stamina_recovery = 0.0f;
     // Recover some stamina every turn.
     // Mutated stamina works even when winded
-    float stamina_multiplier = ( !has_effect( effect_winded ) ? 1.0f : 0.0f ) +
+    float stamina_multiplier = ( !has_effect( effect_winded ) ? 1.0f : 0.1f ) +
```

Comment:
This has a side effect of opening the player up to losing stamina when winded and encumbered, i.e.
stamina_multiplier - 0.1
base regen = 20
mouth encumbrance = 30
on line 4634, stamina_recovery += 0.1 * 20 - 3
stamina_recovery = -1

Can just remove ``if( stamina_multiplier > 0.0f ) {``
and instead clamp stamina_recovery to 0+ after applying encumbrance.

Reply:
The if was redundant and removed, and the negative thing has the max check on it, meaning that it would solve out to a absolute minimum of 0.1 or 1.7 in your example.

Reply:
You are correct, cool.